### ISSUE TITLE
How do I install this mod into the Steam version?

### ISSUE THREAD

**Issue by Hringingy** [Darkroman/FrozenCookies#7]
I would like to use this version of FrozenCookies as it seems to have better logic for Godzamok and FtHoF compared to Icehawk78's version. However there are no instructions for how to install this into Steam

**Comment by Darkroman** [comment]
Honestly I haven't reviewed how to load this into Steam yet. It'll be added into the future! Terribly sorry. :(

**Comment by erbkaiser** [comment]
Hope you can figure it out. Icehawk78's version of Auto-Godmazok is broken in late game and from what I've tested with my save in the web version, your variant performs a lot better.

**Comment by erbkaiser** [comment]
I tried to get the mod to load in Steam but there are too many under-the-hood changes. I now ported over the smart FtHOF and other features to Icehawk's version.
Unfortunately I could not figure out how to do it neatly within Github with commits as the two projects are different.
Perhaps it helps in porting this to Steam.

This version also fixes the broken auto-Godmazok from Icehawk.
https://github.com/erbkaiser/FrozenCookies
